### PR TITLE
[FLINK-3309] [py] Resolve Maven warnings

### PR DESCRIPTION
**Pull request by zentol**

### CODE REVIEW

**Comment by StephanEwen** [comment]
Looks good, +1 to merge

File: flink-libraries/flink-python/src/main/java/org/apache/flink/python/api/PythonPlanBinder.java [hunk 1]
```diff
@@ -432,11 +436,11 @@ private void createPrintSink(PythonOperationInfo info) throws IOException {
...
 
 	private void createBroadcastVariable(PythonOperationInfo info) throws IOException {
-		UdfOperator op1 = (UdfOperator) sets.get(info.parentID);
-		DataSet op2 = (DataSet) sets.get(info.otherID);
+		UdfOperator<?> op1 = (UdfOperator) sets.get(info.parentID);
+		DataSet<?> op2 = (DataSet) sets.get(info.otherID);
 
 		op1.withBroadcastSet(op2, info.name);
-		Configuration c = ((UdfOperator) op1).getParameters();
+		Configuration c = (op1).getParameters();
```
Comment (chiwanpark) [inline flink-libraries/flink-python/src/main/java/org/apache/flink/python/api/PythonPlanBinder.java]: Can we remove parenthesis around `op1`? (`op1.getParameters()`)

Comment (zentol) [inline flink-libraries/flink-python/src/main/java/org/apache/flink/python/api/PythonPlanBinder.java]: sure.

**Comment by zentol** [comment]
Merging this.

**Pull request closed without merging**